pkg/kubelet/cm/cpumanager/policy_static.go

```diff
@@ -79,6 +83,9 @@ type staticPolicy struct {
 	affinity topologymanager.Store
 	// set of CPUs to reuse across allocations in a pod
 	cpusToReuse map[string]cpuset.CPUSet
+	// flag to enable extra allocation restrictions to avoid
+	// different containers to possibly end up on the same core
+	smtAwarenessEnabled bool
```

Comment:
Can we create an `Options` struct to wrap these instead of a single variable for each.